osf/migrations/0146_update_registration_schemas.py

```diff
@@ -31,10 +35,11 @@ def noop(*args, **kwargs):
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('osf', '0146_add_visible_to_registrationschema'),
+        ('osf', '0145_add_visible_to_registrationschema'),
     ]
 
     operations = [
         migrations.RunPython(remove_version_1_schemas, noop),
-        migrations.RunPython(set_invisible_schemas, unset_invisible_schemas),
+        UpdateRegistrationSchemas(),
+        migrations.RunPython(update_v2_schemas, noop),
```

Comment:
@sloria, on second thought, I don't think this migration should have a reverse function. These schemas were invisible/inactive prior to this migration, and a reverse migration shouldn't set them to visible/active.

Reply:
Oh, I see what you mean. Good call.